### ISSUE TITLE
Merited Servers

### ISSUE THREAD

**Issue by IronyTroll** [StrangeLoopGames/EcoSuggestions#1413]
Hello valued team,
TL;DR
-	“Merited Server” validation system.
-	Highlight “Merited Servers” on the main server page.
-	Help new players find established and experienced servers.
-	Recognition for those servers that have been hosting for a long time.
-	Work more closely with servers that have been hosted for ages, not just those who paid the developer fee.

With the recent increase in number of servers and accompanying advertising on multiple platforms of those servers there is a reduction in the overall average citizens on servers. This especially affects those communities/individuals who have invested real-world funds to the purchase of hardware and/or software to facilitate hosting an Eco world for the wider community to play on for a long time. 

We as a community wholeheartedly support the developers interests in allowing anybody to host their own world, albeit through a server host business, or on a dedicated computer in the house, or on the same machine they play on – however I find this to add a great layer of difficulty for the average new player to find a suitable world. There is no distinction to identify which world has an experienced community that has been operating in Eco for the last 10 months, or one that was just hired yesterday by a group of friends who just want to play together. 
My suggestion to SLG is to introduce a “Merited Server” validation system to help citizens, new citizens especially, identify the server hosts with a commitment to server hosting. To the client, this can simply be a secondary list underneath the existing “Official Server” list, called “Merited Servers”. 

These servers can work with SLG more closely and can complement the existing official servers, for example, in geography. There are these committed and experienced servers in USA and Australia. Citizens in Australia rarely play on the official servers due to the 300-500ms ping. Having a Merited Server listed in Australia would not just be a benefit for the server owner, but also SLG through more thorough GitHub issues, feedback, and the citizens, particularly new citizens, who struggle to find new worlds to play on. 

To become a “Merited Server”, SLG may wish to collect personal information of the server owner/admin. They may wish to engage in some tests to ensure the validity of the server, network/hardware tests for example or have the server running for a certain amount of time etc. It is important to recognise “Merited Servers” are NOT official servers, but servers that are recognised for their contribution in sustaining the continuity and growth of Eco by giving them the tick of approval. To say to new players “Hey, this server here has been running a long time, we’ve checked this server, tested them, and they are okay.” 

This “Merited Servers” system NEEDS TO BE COMPLETELY MANUAL. It needs to be checked by an SLG Dev manually and not relegated to a robot. Otherwise, every server will simply become a “Merited Server”. A conversation needs to be had with these admins to assess their level of interest and it needs to be a business decision.

This system is not designed to disenfranchise the creation of new servers, it is simply to award merit to those who have been hosting servers for a long time, who have also contributed to the GitHub, who have included the devs in their worlds, and who have invested real-world resources into accomplishing their contributions and help them achieve their goals of helping this game develop well into the future by helping retain the player base and creating a suitable environment for citizens to join and play.

(Bear in mind, when I talk about long-term worlds, I'm referring to servers that have been hosted for a long time. For example, you could have 30 day meteor worlds, over 8 months, you've had 8 worlds, but you are still a long-term server. Long-term world VS long-term server)

**Comment by BFSexyNurse** [comment]
I’m glad someone is bringing this to peoples attention. It’s vey frustrating to buy a game who’s sole purpose is to build and the server can simply reset every 3-7 days.

**Comment by MishkaFur** [comment]
Would also love a system like this to show the servers who have been around for a considerable time/ran by experienced admins etc, I think this would reward those admins who have been good for the Eco community as well.

I've seen several servers that have been around ages and are well run simply just die out because people are hesistant to join a server thats older than a few days old in most circumstances or just get spammed off the server listing by the huge influx of constantly resetting servers.

I checked the list of servers last night, 53 servers on day 1-3, the categories in the server listing is in general good but when you have a crapload of servers which are all new and based off what ive seen in the help channels recently, quite a few ran by inexperienced admins too it can lead to a not so good experience for a new player needing them to server hop several times until they find a well ran server or just getting frustated from joining a server on day 1 and seeing that the admin has skid steers.

**Comment by johnkslg** [comment]
This is a great suggestion and something I'm actually working on right now. We're adding achievements, and they can be shared among servers that are 'trusted' (aka merited) so you can earn achievements on one server and move to another. We'll be setting up a way for servers to become merited (i actually like that better than trusted as a term) probably with a combination of user voting and manual SLG moderator review.  Then we'll highlight those servers in the server browser.

This will also be tied to accounts, and servers will be associated with SLG accounts, so that if someone gains 'merited' on their last server host, they will keep it, so next time they start a new world they will have 'merited' from the start.

Also adding highlighting in the server browser for servers that have players drop mid-game to get replacements in, and they can get achievements for being 'immigrants' who join servers that need people.

**Comment by IronyTroll** [comment]
Thanks for the response John, I'm aware your team is quite passionate about this. 

I believe that a successful investigation and fix of the in-game browser would compliment this suggestion, but now be a complete solution. (Server Browser - Missing Servers - Filtering #20925) I think this large surge in new (and this, low populated) servers popping up in the last month may in fact be a symptom of this browser issue.
